Index: colorhug-client-0.1.7/libcolorhug/Makefile.am
===================================================================
--- colorhug-client-0.1.7.orig/libcolorhug/Makefile.am
+++ colorhug-client-0.1.7/libcolorhug/Makefile.am
@@ -53,6 +53,7 @@ libcolorhug_la_LIBADD =						\
 	$(GUSB_LIBS)						\
 	$(COLORD_LIBS)						\
 	$(LCMS_LIBS)						\
+	-lgmodule-2.0						\
 	$(NULL)
 
 libcolorhug_la_LDFLAGS =					\
